mm2src/coins/eth.rs

```diff
@@ -215,10 +237,11 @@ impl From<SlurpError> for GasStationReqErr {
     }
 }
 
-#[derive(Debug, Display)]
+#[derive(Debug, Display, EnumFromStringify)]
 pub enum Web3RpcError {
     #[display(fmt = "Transport: {}", _0)]
     Transport(String),
+    #[from_stringify("serde_json::Error")]
     #[display(fmt = "Invalid response: {}", _0)]
     InvalidResponse(String),
```

Comment:
Ive just noticed that Sergey placed the display attribute above the enum type.
![Screenshot 2024-04-09 at 04 05 02](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/16ae95ef-e16c-45da-95e9-e4d49c897269)

However, I agree if many from_stringify variants are at the top of the list, it might become difficult to read. Ive decided to leave the display attribute right above the enumeration type, as types may contain field names. Therefore, placing the display attribute first above the error type makes it easier to see the fields for the error message.

As I understand, we dont have such a rule in the team. Lets just ask other members in our chat which style they prefer. This way, we can start to follow a consistent approach

Reply:
hmm okay either choice is okay. thanks